packages/eslint-plugin/tests/rules/require-img-alt.test.js

```diff
@@ -70,37 +78,85 @@ ruleTester.run("require-img-alt", rule, {
   </body>
 </html>
 `,
-
       errors: [
         {
           messageId: "missingAlt",
           line: 4,
           column: 5,
-          endColumn: 28,
           endLine: 4,
+          endColumn: 28,
+          suggestions: [
+            {
+              messageId: "insertAlt",
+              output: `
+<html>
+  <body>
+    <img src="./image.png" alt="...">
+  </body>
+</html>
+`,
+            },
+          ],
         },
       ],
     },
     {
       code: `
 <html>
   <body>
-    <img src="./image.png">
+    <img src="./image.png" alt="">
```

Comment:
@simplicityf This rule should not report a lint error for `alt=""`. The `alt=""` attribute is useful when the image is purely decorative.